tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -73,7 +73,7 @@ def test_cont_link_flap(self, request, duthost, fanouthosts, bring_up_fanout_int
         # Flap all interfaces one by one on Peer Device
         for iteration in range(3):
             logging.info("%d Iteration flap all interfaces one by one on Peer Device", iteration + 1)
-            candidates = build_test_candidates(duthost, fanouthosts, 'unknown')
+            candidates = build_test_candidates(duthost, fanouthosts, 'all_ports')
```

Comment:
I think I'm missing something - why is this changed from unkown to all_ports?

Reply:
To mute the warning message in candidate list build.